tests/s3/test_multipart_getput.py

```diff
@@ -758,3 +757,102 @@ def test_multipart_upload_test_28528(self):
         self.get_obj_compare_checksums(self.bucket_name, self.object_name, resp[1]["ETag"])
         self.log.info("Stop and validate parallel S3 IOs")
         self.log.info("ENDED: Test List multipart with 10000 parts")
+
+    @pytest.mark.tags('TEST-40255')
+    @pytest.mark.s3_ops
+    @CTFailOn(error_handler)
+    def test_multipart_upload_test_40255(self):
+        """
+        This test is for overwriting multipart uploaded object by simple object
+        """
+        mp_config = MPART_CFG["test_28539"]
+        s3_background_io = S3BackgroundIO(s3_test_lib_obj=self.s3_test_obj)
```

Comment:
I also see that  multipart upload step in this test can be extracted to another function and used.  I also don't understand the use of background IOs. Per me they are irrelevant in this context.

Reply:
Background IOs are just to make sure, that they are undisturbed despite any multipart operation happens